src/index.js

```diff
@@ -0,0 +1,111 @@
+import { registerPlugin } from '@wordpress/plugins';
+import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-site';
+import { blockDefault } from '@wordpress/icons';
+import { Fragment, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import { Button, MenuGroup, MenuItem, PanelBody, TextControl } from '@wordpress/components';
+import { store as noticesStore } from '@wordpress/notices';
+import { useDispatch } from '@wordpress/data';
+
+const GlobalStylesProvider = wp.editSite.GlobalStylesProvider;
+
+const StyleVariations = () => {
+	const [ variationName, setVariationName ] = useState( '' );
+	const [ canReset, onReset ] = wp.editSite.useGlobalStylesReset();
+	const { createErrorNotice } = useDispatch( noticesStore );
+
+	async function createVariation() {
+		try {
+			const response = await apiFetch( {
+				path: '/create-block-theme/v1/variation/' + variationName,
+			} );
+
+			// Clear global styles customizations
+			onReset();
+
+			// Reload variations
+			// We need to invalidate and maybe refetch __experimentalGetCurrentThemeGlobalStylesVariations(),
+			if ( response.req.status === 200 ) {
+				createErrorNotice( __( 'Variation created successfully' ), { type: 'snackbar' } );
+			}
+
+		} catch ( error ) {
+			const errorMessage =
+				error.message && error.code !== 'unknown_error'
+					? error.message
+					: __( 'An error occurred while creating the site export.' );
+			createErrorNotice( errorMessage, { type: 'snackbar' } );
+		}
+	}
+
+	return (
+		<PanelBody>
+			<MenuGroup label={ __( 'Create style variation' ) }>
```

Comment:
There's a padding applied to this label that I think looks off, I think it should be just left aligned, if possible. Maybe that's a problem with the component though.

<img width="278" alt="Screen Shot 2022-08-09 at 2 25 04 PM" src="https://user-images.githubusercontent.com/5375500/183734301-1962615c-982d-49e9-9aef-03893b834b85.png">

Reply:
Yeah I'd rather fix that in the component itself.